Readme.md

```diff
@@ -76,17 +82,19 @@ AadhaarAPI | Zoop IOS SDK for Income Tax Return and Bank Statement Analysis Gate
 8. [BIOMETRIC DEVICES SETUP](#esignBiometric)
 9. [PULLING TRANSACTION STATUS AT BACKEND](#esignStatus)
    - [RESPONSE PARAMS](#esignStatusResp)
-## Zoop.one Income Tax Return(ITR) Gateway
+   
+## Zoop.one Income Tax Department V2.0(ITD) Gateway (Beta)
 
 <a name="itrIntroduction"></a>
 ### 1. INTRODUCTION
-**Income Tax Return** is the form in which assessee files information about his Income and tax thereon to Income Tax Department. The Zoop ITR Gateway allows allows you to fetch the tax returns of your client and make better business decision on them.
+**Income Tax Department** holds and verifies the earnings and tax related information of an individual.
+Income tax return is a form where a person submits information about his income earned and the tax paid to the Department. Whereas 26AS is a consolidated annual tax credit statement of an individual's earnings as per the ITD records. The Zoop ITD Gateway allows you to fetch and verify the financial information of your client in order to make better business decisions.
 
 <a name="itrProcessFlow"></a>
 ### 2. PROCESS FLOW
-1. At your backend server, Initiate the **ITR** transaction using a simple Rest API [POST] call. Details of these are available in the documents later. You will require API key and Agency Id for accessing this API which can be generated from the Dashboard.
-2. This gateway transaction id then needs to be communicated back to the frontend(in Android Project) where SDK is to be called.
-3. After adding the Framework in your ios project, client has to pass the above generated transaction id to a Framework View Controller, which will open the Framework.
+1. At your backend server, Initiate the **ITD** transaction using a simple Rest API [POST] call. Details of these are available in the documents later. You will require API key and Agency Id for accessing this API which can be generated from the Dashboard.
+2. This gateway transaction id then needs to be communicated back to the frontend(in IOS Project) where SDK is to be called.
+3. After adding the Framework in your IOS project, client has to pass the above generated transaction id to a Framework View Controller, which will open the Framework.
```

Comment:
Shouldn't be iOS instead?

Reply:
ya but that's fine I guess don't think it's a major issue will change in next pull request.